code/modules/unit_tests/organs.dm

```diff
@@ -0,0 +1,174 @@
+/**
+ * Organ Unit Tests
+ *
+ * Test 1. organ_sanity:
+ * Checks all implantable organs.
+ * Ensures algorithmic correctness of the "Insert()" and "Remove()" procs.
+ * This test is especially useful because developers frequently  override those.
+ *
+ * Test 2. organ_set_bonus_id:
+ * Checks that all "organ_set_bonus" status effects have unique "id" vars.
+ * Required to ensure that the status effects are treated as "unique".
+ *
+ * Test 3. organ_set_bonus_sanity
+ * Checks that all implantable DNA Infuser organs are set up correctly and without error.
+ * Ensures that the "organ_set_bonus" status effects activate and deactivate.
+ *
+ * "Life's about having a good time and having candy, not putting roaches in your hair!"
+ * - Dr. Steve Brule
+ */
+/// Ensures the organ inserted properly by validating specific variables.
+#define TEST_ORGAN_INSERTED_OK(lab_rat, test_organ) ((test_organ.owner == lab_rat) && (test_organ in lab_rat.internal_organs) && test_organ.slot ? (lab_rat.getorganslot(test_organ.slot) == test_organ) : TRUE)
+/// Ensures the organ removed properly by validating specific variables.
+#define TEST_ORGAN_REMOVED_OK(lab_rat, test_organ) ((test_organ.owner == null) && !(test_organ in lab_rat.internal_organs) && test_organ.slot ? (lab_rat.getorganslot(test_organ.slot) != test_organ) : TRUE)
```

Comment:
These are unreadable and we get zero benefit from this being a macro, please turn these into normal procs.